### PR TITLE
Use Cache-Control for images

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -23,5 +23,8 @@
     "deno-bin": ">=1.25.2",
     "serve": "^14.1.2",
     "vuepress-vite": "2.0.0-beta.60"
+  },
+  "dependencies": {
+    "glob": "^8.1.0"
```
Comment (KnorpelSenf) [inline site/package.json:28]: Why is this necessary?

Comment (roj1512) [reply site/package.json:28]: Mistake

**Review by KnorpelSenf — APPROVED** [review]
LGTM, thanks! Anything left to do here?

**Comment by roj1512** [comment]
Merged before the build passing hence #619
